_layouts/default.html

```diff
@@ -49,5 +49,11 @@
     <script src="/semantic/dist/semantic.min.js"></script>
     <script src="/js/jquery.scrollie.min.js"></script>
     <script src="/js/scripts.js?1"></script>
+    <script>
+      $('.modal-open').click(function(e) {
+        e.preventDefault();
+        $('.ui.modal').modal('show');
+      });
+    </script>
```

Comment:
This is not great but I figured it would make sense to say that any button tagged with the `modal-open` class will be used to open any modals on the same page. I would have liked to keep all the JavaScript related to my awful modal contained within the sessions page; however, as jQuery is included below the main body, I wouldn't have been able to call it without putting jQuery in the `head`.